features/cellular/framework/AT/ATHandler.cpp

```diff
@@ -893,6 +896,8 @@ void ATHandler::resp_stop()
     set_tag(&_resp_stop, OK);
     // Reset info resp prefix
     memset(_info_resp_prefix, 0, sizeof(_info_resp_prefix));
+
+    _last_response_stop = rtos::Kernel::get_ms_count();
```

Comment:
Should be 0 to avoid redundant initial waiting.

Reply:
Set 0 in the constructor